lib/parameters.py

```diff
@@ -343,3 +343,18 @@ def gen_vpp_rotation_parameters(spec):
   keys = ("case", "degrees")
   params = gen_vpp_rotation_variants(spec)
   return keys, params
+
+
+def gen_transcode_variants(spec,codec,ttype):
+  for case, params in spec.iteritems():
+    variants = params.get("one_one", None) or []
+    for variant in variants:
+      variant_ttype=variant.get("transcode_type",None) or ['hwhw']
```

Comment:
`transcode_type` should be a single value, but your fallback/default value is a list.

Reply:
fallback removed